js/graviex.js

```diff
@@ -0,0 +1,1073 @@
+        //     "volume2": "0.03837782973",
+        //     "sell": "0.00000058",
+        //     "buy": "0.00000055",
+        //     "at": 1643628427,
+        // },
+        return this.parseTicker (response, market);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const response = await this.publicGetTickers (params);
+        // {
+        //   "giobtc": {
+        //     "name": "GIO/BTC",
+        //     "base_unit": "gio",
+        //     "base_fixed": 4,
+        //     "base_fee": 0.002,
+        //     "quote_unit": "btc",
+        //     "quote_fixed": 8,
+        //     "quote_fee": 0.002,
+        //     "api": true,
+        //     "base_lot": null,
+        //     "quote_lot": null,
+        //     "base_min": "0.00000010",
+        //     "quote_min": "0.00000010",
+        //     "blocks": 260722,
+        //     "block_time": "2022-01-31 14:10:08",
+        //     "wstatus": "on",
+        //     "low": "0.0000005",
+        //     "high": "0.00000058",
+        //     "last": "0.00000058",
+        //     "open": "0.000000510",
+        //     "volume": "70673.0412",
+        //     "volume2": "0.03837782973",
+        //     "sell": "0.00000057",
+        //     "buy": "0.00000055",
+        //     "at": 1643628325,
+        //   },
+        // },
+        // const data = response;
+        // let timestamp = data['at'];
+        // let tickers = data['ticker'];
+        const ids = Object.keys (response);
+        const result = {};
+        for (let i = 0; i < ids.length; i++) {
+            const id = ids[i];
+            const market = this.safeMarket (id);
+            const symbol = market['symbol'];
+            result[symbol] = this.parseTicker (response[id], market);
+        }
+        return result;
+    }
+
+    parseTrade (trade, market = undefined) {
+        // {
+        //     "id": 14473952,
+        //     "at": 1643681026,
+        //     "price": "0.00000056",
+        //     "volume": "0.2",
+        //     "funds": "0.000000112",
+        //     "market": "giobtc",
+        //     "created_at": "2022-02-01T05:03:46+03:00",
+        //     "side": "sell"
+        // },
+        const price = this.safeString2 (trade, 'p', 'price');
+        const amount = this.safeString2 (trade, 'volume', 'amount');
+        const funds = this.safeString (trade, 'funds');
+        const marketId = this.safeString (trade, 'market');
+        const symbol = this.safeSymbol (marketId, market);
+        const datetime = this.safeValue (trade, 'created_at');
+        let timestamp = this.parse_date (datetime);
+        let id = this.safeString2 (trade, 't', 'a');
+        id = this.safeString2 (trade, 'id', 'tid', id);
+        const side = this.safeValue (trade, 'side');
+        let takerOrMaker = undefined;
+        const orderId = this.safeString (trade, 'id');
+        if ('side' in trade) {
+            if (trade['side'] === 'buy') {
```

Comment:
let's use `side` instead of `trade['side']` here

Reply:
OKay. Fixed in the latest commit https://github.com/ccxt/ccxt/commit/24b2627beff01a76b81a32b5728efb6846cd8010